src/sentry/api/serializers/models/project_platform.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+from sentry.api.serializers import register, Serializer
+from sentry.models import ProjectPlatform
+
+
+@register(ProjectPlatform)
+class ProjectPlatformSerializer(Serializer):
+    """
+    Tracks usage of a platform for a given project.
```

Comment:
The only thing I'd say is this comment doesn't really apply on this serializer. I'd just remove it. It's obvious that it's a serializer for the ProjectPlatform model, and it doesn't need to repeat exactly the same comment.